src/include/btmem.h

```diff
@@ -166,9 +166,10 @@ struct __wt_delta_header {
 #define WT_DELTA_HEADER_SIZE 20
 
 /*
- * The number of deltas for a base page must be strictly less than WT_DELTA_LIMIT. Changes past that
- * point are reflected by making a new base page. Thus WT_DELTA_LIMIT can be used to size arrays
- * that contain the base page plus all associated deltas.
+ * The number of deltas for a base page must be strictly less than or equal to WT_DELTA_LIMIT.
+ * Though we have made the value adjustable, 32 remains the maximum value we support. Changes past
+ * that point are reflected by making a new base page. Thus WT_DELTA_LIMIT can be used to size
+ * arrays that contain the base page plus all associated deltas.
```

Comment:
I think this comment should also point out that there's config to adjust if the limit is changed.